packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -202,13 +223,37 @@ export class Coachmark extends BaseComponent<ICoachmarkTypes, ICoachmarkState> {
                     color={color}
                   />
                 )}
-                <FocusZone>
-                  <div className={classNames.entityHost} data-is-focusable={true} onFocus={this._onFocusHandler}>
-                    <div className={classNames.entityInnerHost} ref={this._entityInnerHostElement}>
+                <FocusTrapZone isClickableOutsideFocusTrap={true} forceFocusInsideTrap={false}>
+                  <div
+                    className={classNames.entityHost}
+                    tabIndex={-1}
+                    data-is-focusable={true}
+                    onClick={this._onFocusHandler}
+                    role="dialog"
+                    aria-labelledby={ariaLabelledBy}
+                    aria-describedby={ariaDescribedBy}
+                  >
+                    {isCollapsed &&
+                      ariaLabelledBy && (
+                        <h1 id={ariaLabelledBy} className={classNames.ariaContainer}>
```

Comment:
why H1? I thought we decided to not use a header

Reply:
Changed to a `<p`> element